LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Modified BSD License
```

Comment:
Have we actually settled on a license yet?

I guess we should put *something* in here. But this hasn't been approved by anybody...